src/routes/+page.ts

```diff
@@ -0,0 +1,16 @@
+import { redirect } from '@sveltejs/kit';
+import { get } from 'svelte/store';
+
+export async function load({ parent }) {
+	const { game } = await parent();
+	const $game = get(game);
+
+	if (
+		!$game?.player1.name ||
+		!$game?.player1.handicap ||
+		!$game?.player2.name ||
+		!$game?.player2.handicap
```

Comment:
can we get away with this simpler condition instead?
```suggestion
		!$game
```